interface-definitions/service_monitoring_telegraf.xml.in

```diff
@@ -85,6 +85,89 @@
                 </properties>
                 <defaultValue>all</defaultValue>
               </leafNode>
+              <node name="prometheus-client">
+                <properties>
+                  <help>Output plugin Prometheus client</help>
+                </properties>
+                <children>
+                  <node name="authentication">
+                    <properties>
+                      <help>HTTP basic authentication parameters</help>
+                    </properties>
+                    <children>
+                      <leafNode name="login">
```

Comment:
Would renaming it to username make sense?

Reply:
Done!